src/_core/Logger.js

```diff
@@ -1,6 +1,10 @@
 class Logger {
   constructor() {
-    this.debugMode = true;
+    this.debugMode = !("update_url" in chrome.runtime.getManifest());
+  }
+
+  toggleDebug() {
```

Comment:
What is the purpose of this function?

Reply:
I had originally planned to make a debug and clear storage button somewhere in a development tab of the extension UI, but the storage changes ended up redirecting me. This function will allow doing so in the future.